docs/quick_start.md

```diff
@@ -8,9 +8,21 @@ To start a new task locally, in development mode follow these steps:
 
 ### 1. Start your new task from our template repository
 
-The simplest way to get started is creating a new repository using Honeycomb as a template.
+The simplest way to get started is creating a new repository using Honeycomb as a template. This option will allow you to generate a new repository with the same directory structure and files as an existing repository.
 
-Go to https://github.com/brown-ccv/honeycomb and click on `Use this template` on the top right. Then select the organization and the name of your repository and click on `create repository from template`
+Creating a repository from a template is similar to forking a repository, but there are important differences:
+
+- A new fork includes the entire commit history of the parent repository, while a repository created from a template starts with a single commit.
+- Commits to a fork don't appear in your contributions graph, while commits to a repository created from a template do appear in your contribution graph.
+- A fork can be a temporary way to contribute code to an existing project, while creating a repository from a template starts a new project quickly.
+
+Go to https://github.com/brown-ccv/honeycomb and click on `Use this template` on the top right. 
+
+![](assets/template.png)
+
+Then select the organization and the name of your repository and click on `create repository from template`.
+
+![](assets/new_task.png)
 
 Alternatively, you can use GitHub CLI to create a new project based on the Honeycomb template repository. First, install GitHub CLI (https://cli.github.com/), then simply run on your terminal: 
```

Comment:
Looks like this was in the original markdown but "just", "simply", etc  shouldn't be in docs. Can invoke some imposters syndrome

```suggestion
Alternatively, you can use GitHub CLI to create a new project based on the Honeycomb template repository. First, install GitHub CLI (https://cli.github.com/), then run on your terminal: 
```